docs/developer_documentation/create_assistant/packaging_and_distributing.rst

```diff
@@ -156,3 +157,59 @@ re-run `da tweak dap pack` and `da pkg lint foo-0.0.2.dap`, and upload it to
 DAPI just the same. The server will understand it's an update and will act
 accordingly.
 
+Uploading from command line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you'd prefer to upload your package via command line, you can use ``extra`` Assistant from the
+dap DAP mentioned :ref:`before <dap_dap>`::
+
+  # assuming you dap is packaged in foo-0.0.1.dap
+  $ da extra dap upload -d foo-0.0.1.dap
+
+When you do this for the first time, you will be prompted for your authorization token.
+To obtain it, log into `DAPI <http://dapi.devassistant.org/>`_ with your Github
+or Fedora account, click on your username in the top menu and select *View profile*.
+Get the token on the bottom of your profile page.
+
+Note that by uploading the dap via command line, you agree to the `DAPI Terms of Use
+<https://dapi.devassistant.org/terms/>`_.
```

Comment:
I'm not sure if we're doing it, so please disregard this comment if we do: Would it be worth it to show this link to DAPI Terms in the CLI when uploading too? At least the first time the user uploads a DAP.